lib/linux_admin/registration_system/subscription_manager.rb

```diff
@@ -34,7 +34,7 @@ def organizations(options)
       params.merge!(proxy_params(options))
 
       result = run!(cmd, :params => params)
-      parse_output(result.output).index_by {|i| i[:name]}
+      parse_output(result.output).each_with_object({}) { |i, h| h[i[:name]] = i }
```

Comment:
Was about to suggest using `.index_by`...LOL...I would have failed that question if it came up on http://railshurts.com/quiz/